test/verify/check-system-info

```diff
@@ -34,6 +34,10 @@ VERSION_ID="2.0"
 PRETTY_NAME="Foobar Adventure Linux Server 2.0 (Day of Doom)"
 """
 
+lscpu = """
+#!/bin/sh
```

Comment:
Please drop the initial newline. Also, please move this into the test case, it's a bit too global here. We might also have more scenarios in the future.